posix.go

```diff
@@ -439,7 +439,15 @@ func (s fsStorage) CreateFile(volume, path string) (writeCloser io.WriteCloser,
 	filePath := pathJoin(volumeDir, path)
 	// Verify if the file already exists and is not of regular type.
 	var st os.FileInfo
-	if st, err = os.Stat(filePath); err == nil {
+	if st, err = os.Stat(filePath); err != nil {
+		// File path segment is too long than supported length
+		if strings.Contains(err.Error(), "file name too long") {
+			return nil, errFileNameTooLong
+		} else if strings.Contains(err.Error(), "system cannot find the path specified") {
```

Comment:
This error is because the file doesn't exist "system cannot find the path specified" is "os.IsNotExist(err)" . did you validate this directly on windows node?